x11-libs/wxGTK/wxGTK-3.0.3.1.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit multilib-minimal
+
+DESCRIPTION="GTK+ version of wxWidgets, a cross-platform C++ GUI toolkit"
+HOMEPAGE="https://wxwidgets.org/"
+
+# new wxGTK releases can be found from github repository for wxWidgets
+SRC_URI="https://github.com/wxWidgets/wxWidgets/releases/download/v${PV}/wxWidgets-${PV}.tar.bz2
+	doc? ( https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.3/wxWidgets-3.0.3-docs-html.tar.bz2 )"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sh ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+IUSE="+X aqua doc debug gstreamer libnotify opengl sdl tiff webkit"
+
+SLOT="3.0"
+
+RDEPEND="
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	sdl?    ( media-libs/libsdl[${MULTILIB_USEDEP}] )
+	X?  (
+		>=dev-libs/glib-2.22:2[${MULTILIB_USEDEP}]
+		media-libs/libpng:0=[${MULTILIB_USEDEP}]
+		sys-libs/zlib[${MULTILIB_USEDEP}]
+		virtual/jpeg:0=[${MULTILIB_USEDEP}]
+		>=x11-libs/gtk+-2.18:2[${MULTILIB_USEDEP}]
+		x11-libs/gdk-pixbuf[${MULTILIB_USEDEP}]
+		x11-libs/libSM[${MULTILIB_USEDEP}]
+		x11-libs/libXxf86vm[${MULTILIB_USEDEP}]
+		x11-libs/pango[X,${MULTILIB_USEDEP}]
+		gstreamer? (
+			media-libs/gstreamer:1.0[${MULTILIB_USEDEP}]
+			media-libs/gst-plugins-base:1.0[${MULTILIB_USEDEP}] )
+		libnotify? ( x11-libs/libnotify[${MULTILIB_USEDEP}] )
+		opengl? ( virtual/opengl[${MULTILIB_USEDEP}] )
+		tiff?   ( media-libs/tiff:0[${MULTILIB_USEDEP}] )
+		webkit? ( net-libs/webkit-gtk:2 )
+		)
+	aqua? (
+		x11-libs/gtk+:2[aqua=,${MULTILIB_USEDEP}]
+		virtual/jpeg:0=[${MULTILIB_USEDEP}]
+		tiff?   ( media-libs/tiff:0[${MULTILIB_USEDEP}] )
+		)"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig[${MULTILIB_USEDEP}]
+	opengl? ( virtual/glu[${MULTILIB_USEDEP}] )
+	X?  (
+		x11-proto/xproto[${MULTILIB_USEDEP}]
+		x11-proto/xineramaproto[${MULTILIB_USEDEP}]
+		x11-proto/xf86vidmodeproto[${MULTILIB_USEDEP}]
+	)"
+
+PDEPEND=">=app-eselect/eselect-wxwidgets-20131230"
+
+LICENSE="wxWinLL-3
+		GPL-2
+		doc?	( wxWinFDL-3 )"
+
+S="${WORKDIR}/wxWidgets-${PV}"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-3.0.0.0-collision.patch
```

Comment:
unclear if collision patch is working correctly.

eselect module possibly needs updated as well